Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -909,6 +922,12 @@ open class ComposerVC: _ViewController,
         guard let maxMessageLength = channelConfig?.maxMessageLength else { return true }
         return textView.text.count + (text.count - range.length) <= maxMessageLength
     }
+    
+    open func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
+        let detector = TextViewMentionedUserDetector()
```

Comment:
This should be a reference property so that it can be easy replaced by the customer and in tests.
```suggestion

```